src/controllers/monitoring/alert/index.ts

```diff
@@ -21,6 +22,57 @@ const updateAlertState = async (params) => {
     return response;
 };
 
+const changeAlertState = async (params) => {
+    if (!params.alerts) {
+        throw new Error('Required Parameter. (key = alerts)');
+    }
+
+    if (!params.state) {
+        throw new Error('Required Parameter. (key = state)');
+    } else if (['ACKNOWLEDGED', 'RESOLVED'].indexOf(params.state) < 0) {
+        throw new Error('Invalid Parameter. (state = ACKNOWLEDGED | RESOLVED)');
+    }
+
+    const monitoringV1 = await grpcClient.get('monitoring');
+
+    let successCount = 0;
+    let failCount = 0;
+    const failItems = {};
+
+    const promises = params.alerts.map(async (alert_id) => {
+        try {
+            const reqParams = {
+                alert_id: alert_id,
+                state: params.state,
+                ... params.domain_id && {domain_id : params.domain_id}
+            };
+
+            await monitoringV1.Alert.update(reqParams);
+            successCount = successCount + 1;
+        } catch (e) {
+            failItems[alert_id] = e.details || e.message;
+            failCount = failCount + 1;
+        }
+    });
+    await Promise.all(promises);
+
+    if (failCount > 0) {
+        const error: ErrorModel = new Error(`Failed to change alerts. (success: ${successCount}, failure: ${failCount})`);
+        error.fail_items = failItems;
+        throw error;
+    } else {
+        return {};
+    }
+};
+
+const changeToAcknowledge = async (params) => {
+    await changeAlertState(params);
+};
+
+const changeToResolve = async (params) => {
+    await changeAlertState(params);
+};
+
```

Comment:
제 생각에는 acknowledge/resolve api 는 스펙 자체가 update state  와 완전히 달라야 할 것 같아요.
위와 같이 하면, 함수를 나누는 의미가 전혀 없어보입니다.

params spec 자체를
```
{
  alert_ids: string[];
}
```

이렇게 주고, 각각의 api function에서 acknowledge/resolve 로 상태를 변경해주면 좋을 것 같습니다.

예를 들면 아래처럼요.

```suggestion
interface ChangeStateParams = {
  alert_ids: string[];
}

const changeToAcknowledge = async (params: ChangeStateParams) => {
 if (!Array.isArray(parmas.alert_ids)) {...}
 
    await changeAlertState(params, 'ACKNOWLEDGED');
};

const changeToResolve = async (params: ChangeStateParams) => {
 if (!Array.isArray(parmas.alert_ids)) {...}
 
    await changeAlertState(params, 'RESOLVED');
};

```

Reply:
change-state api 로 통일하기로 하여 위 코멘트는 무시해주시기 바랍니다.